app/src/test/java/at/aau/edu/wizards/TooManyPlayerUnitTest.kt

```diff
@@ -0,0 +1,66 @@
+package at.aau.edu.wizards
```

Comment:
Please move the package to the class you want to test. Checkout some of the `api.impl` tests otherwise please